webpack/assets/javascripts/react_app/components/TemplateGenerator/TemplateGeneratorSelectors.js

```diff
@@ -0,0 +1,15 @@
+import { createStructuredSelector } from 'reselect';
+
+const selectTemplates = state => state.templates;
+const polling = state => selectTemplates(state).polling;
+const dataUrl = state => selectTemplates(state).dataUrl;
+const generatingError = state => selectTemplates(state).generatingError;
+const generatingErrorMessages = state =>
+  selectTemplates(state).generatingErrorMessages;
+
+export const selectGeneratingProps = createStructuredSelector({
```

Comment:
using reselct here is redundant because no calculations have made here, so memorization won't be effective.
simple selectors would be enough.
Plus. could you add a test? we have a test helper for selectors as well.

Reply:
:dart: thanks, I thought the `createStructuredSelector` is meant to take out the boilerplate of combined selector, but it does memorization as well.
I believe I don't need the helper here as my selector is really simple and I believe my test is as well, I have added one.